Vagrantfile

```diff
@@ -19,6 +19,7 @@ echo 'cd ~/go/src/github.com/cilium/cilium' >> /home/vagrant/.bashrc
 SCRIPT
 
 $build = <<SCRIPT
+go get -u github.com/jteeuwen/go-bindata/...
```

Comment:
Be aware this will be run every time someone reloads the VM. I think this can be moved to `$bootstrap` since that repository hasn't received any commits since 2015. And there's no need for the `-u` in `$bootstrap`.